mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIntegrationTest.java

```diff
@@ -80,6 +84,20 @@
     @Rule
     public RuleChain ruleChain = RuleChain.outerRule(temporaryFolder).around(embeddedElasticSearch);
 
+    @ClassRule
+    public static TikaContainer tika = new TikaContainer();
```

Comment:
why do you need tika, you changed nothing else in the tests and it worked before

Reply:
The previous TextExtractor doesn't support PDF extracting (only text/* content-types).
So I introduce Tika for the addition of a test.